src/components/views/messages/MLocationBody.tsx

```diff
@@ -115,11 +120,25 @@ export function LocationBodyContent(props: ILocationBodyContentProps) {
                 </div>
                 : null
         }
-        <div
-            id={props.bodyId}
-            onClick={props.onClick}
-            className="mx_MLocationBody_map"
-        />
+        {
+            props.tooltip
+                ? <TooltipTarget
+                    label={props.tooltip}
+                    alignment={Alignment.InnerBottom}
+                    maxParentWidth={450}
+                >
+                        <div
```

Comment:
```suggestion
                    <div
```

Reply:
Fixed in 04aebeb0e0ed975a45f49191f8830fca949bda45